shared/searchv3/user-input/index.js.flow

```diff
@@ -0,0 +1,22 @@
+// @flow
+import {Component} from 'react'
+import * as Constants from '../../constants/searchv3'
+import type {IconType} from '../../common-adapters/icon'
+
+export type UserDetails = {
+  followingState: Constants.FollowingState,
```

Comment:
I added `id` to this, since a username is different than the id. e.g.: username:marcopolo, id:marcopolo@github